setup.py

```diff
@@ -22,9 +22,7 @@
               , 'Topic :: Internet :: WWW/HTTP :: WSGI :: Application'
                ]
 
-setup( author = 'Gratipay, LLC'
-     , author_email = 'support@gratipay.com'
```

Comment:
Are you sure we can get away with no author?

> Either the author or the maintainer must be identified.

https://docs.python.org/2/distutils/setupscript.html#additional-meta-data